src/components/BreadCrumb/BreadCrumb.tsx

```diff
@@ -0,0 +1,66 @@
+import { Breadcrumb, BreadcrumbItem, PageBreadcrumb } from '@patternfly/react-core';
+import { isUndefined } from 'lodash';
+import { useMatches } from 'react-router-dom';
+
+import { TBreadCrumbAttributes, breadCrumbAttributes } from 'common/breadCrumbAttributes';
+
+import { uiLogger } from 'services/uiLogger';
+
+export interface ISingleBreadCrumb {
+  entity: string;
+  data: string | undefined;
+}
+
+export interface IBreadCrumbProps {
+  pageBreadCrumbs: ISingleBreadCrumb[];
+}
+
+export const BreadCrumb = ({ pageBreadCrumbs }: IBreadCrumbProps) => {
+  const matches = useMatches();
+  const matchCrumbs = matches.filter((match) => !isUndefined(match.handle));
+  const pageCrumbs = pageBreadCrumbs as ISingleBreadCrumb[];
+
+  const breadCrumbs = matchCrumbs.map((crumb, index) => {
+    const notLast = index < matchCrumbs.length - 1;
+    if (crumb.handle && typeof crumb.handle === 'string') {
```

Comment:
`crumb.handle` will always be defined here (because of the filter above)
this pattern is preferable:
```
if(typeof crumb.handle !== 'string') {
  uiLogger.log("Breadcrumb route 'handle' is not string, this option is not supported.");
  return [];
}

...
```